### PR TITLE
Group outbound messages by phone number and send in batch to queue

### PR DESCRIPTION
**Pull request by jeffreysilver**
The send message lambda will receive an array of messages and will be in charge of sleeping for some amount of time between sending each one to twilio. Note that we arent including the `with_initial_pause` argument from the simulator. Im going to do some heavy testing to see if its needed with the way messages are getting bundled.

### CODE REVIEW

File: stopcovid/clients/sqs.py [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
-from typing import List
 
+from typing import List
+from collections import defaultdict
 from stopcovid.event_distributor.outbound_sms import OutboundSMS
 
 
+def _get_message_deduplication_id(messages):
+    unique_message_ids = sorted(list(set([str(message.event_id) for message in messages])))
+    return "-".join(unique_message_ids)
```
Comment (pdsarin) [inline stopcovid/clients/sqs.py:12]: this works, though we'll soon have a batch ID that we can use

Comment (pdsarin) [reply stopcovid/clients/sqs.py:12]: ignore. You're potentially aggregating across batches.

Comment (jeffreysilver) [reply stopcovid/clients/sqs.py:12]: Do you think that the outbound sms flow should know about event batches at all? We flatten them out into events slightly upstream. I'll stew on it

Comment (pdsarin) [reply stopcovid/clients/sqs.py:12]: probably not... Agree with your instinct.

**Pull request merged**